.eslintrc

```diff
@@ -33,9 +33,7 @@
     "@typescript-eslint/typedef": ["error", { "parameters": true }],
     "prefer-const": 1,
     "radix": 1,
-    "react/jsx-uses-vars": 0, // Project does not use react
-    "react/no-deprecated": 0,
-    "react/prop-types": 0,
+    "react/prop-types": 0, // This relies specifically on React PropTypes and ignores other typing methods
```

Comment:
I think all these changes make sense except for why remove "react/no-deprecated" ?

Reply:
This removes the rule that disables the no-deprecated check! So in this case it's actually enabling it :)